tools/test/examples/examples_lib.py

```diff
@@ -19,7 +19,8 @@
 from tools.export import EXPORTERS
 
 SUPPORTED_TOOLCHAINS = ["ARM", "IAR", "GCC_ARM"]
-SUPPORTED_IDES = ["iar", "uvision", "make_gcc_arm", "make_iar", "make_armc5"]
+SUPPORTED_IDES = [exp for exp in EXPORTERS.keys() if exp != "cmsis" and exp != "zip"]
```

Comment:
Can we introspect the class then to see if there's a build function and filter based on that?